encoding.bs

```diff
@@ -1332,6 +1335,7 @@ requires buffering of scalar values.
 
 <p>The <dfn constructor for=TextEncoder id=dom-textencoder><code>TextEncoder()</code></dfn>
 constructor, when invoked, must return a new {{TextEncoder}} object.
+<!-- TODO: what should this say in the new world? -->
```

Comment:
Interesting. TextEncoder has no state so there's nothing to set up here. I guess this points to the fact that it really shouldn't be a class (it's just behavior, not data + behavior with individual instances having some meaning). But it is, presumably for symmetry, so here we are.

I guess "must do nothing"?

Reply:
Do we still want to say `TextEncoder()` as a heading? Or should that also change somehow? (It could be considered a class that has the UTF-8 encoder as internal thing btw. But the spec grabs that as a global resource.)